pkg/apis/build/v1alpha1/builder_resource.go

```diff
@@ -8,4 +8,5 @@ type BuilderResource interface {
 	Ready() bool
 	BuildpackMetadata() corev1alpha1.BuildpackMetadataList
 	RunImage() string
+	GetKind() string
```

Comment:
this should probably only need to be in the `v1alpha2` package

Reply:
Where is the `GetKind()` method called? Is it necessary?